types/landing.ts

```diff
@@ -3,5 +3,5 @@ import React from "react";
 export interface LandingCardProps {
   title: string;
   subTitle: string;
-  children: React.SVGProps<SVGSVGElement>;
+  children: JSX.Element | JSX.Element[];
```

Comment:
이 부분이 이해가 안돼.. 지금 보니까 타입 컨플릭트 난다고 안내가 뜨긴하네.. ReactNode로도 안되나?! 낼 모레 풀 받아서 해봐야겠당

```
Use any to avoid conflicts with @svgr/webpack plugin or babel-plugin-inline-react-svg plugin.
```